current/runtime/fees.md

```diff
@@ -4,118 +4,151 @@ lang: en
 
 ```rust
 #[weight = SimpleDispatchInfo::FixedNormal(10)]
 pub fn some_normal_function_heavy() { some_computation(); }
 ```
 
-### Fixed Operational
+#### `MaxNormal`
+
+This is equivalent to `SimpleDispatchInfo::FixedNormal(Weight::max_value())`.
 
-This function will have a fixed weight but can consume the reserved operational portion as well.
+#### `InsecureFreeNormal`
+
+This means that the function has no weight; it will not contribute to block fullness at all and no
+weight-fee is applied. Although the `base_fee` and `length_fee` still need to be paid, as the name
+indicates a lack of weight fees also implies a lack of security, so dispatches of this type should
+be used with care.
 
 ```rust
-#[weight = SimpleDispatchInfo::FixedOperational(20)]
-pub fn mission_critical_function() { some_sudo_op(); }
+#[weight = SimpleDispatchInfo::InsecureFreeNormal]
+pub fn some_normal_function_light() { noop(); }
 ```
 
-### Default
+### Operational Dispatches
+
+Operational dispatches are those that go beyond user-triggered transactions; they are dispatches
+that are necessary to provide network capabilities. These types of dispatches are not bound by the
+[`AvailableBlockRatio`](https://substrate.dev/rustdocs/master/frame_system/trait.Trait.html#associatedtype.AvailableBlockRatio),
+which means they may consume a block's entire weight limit. Dispatches in this class are given maximum
+priority and are exempt from paying the `base_fee` and `length_fee`.
+
+#### `FixedOperational`
 
-This function will automatically get `#[weight = SimpleDispatchInfo::default()]`.
+Describes a function that will have a fixed weight and can consume the reserved operational portion
+as well.
 
 ```rust
-pub fn something_else() { noop(); }
+#[weight = SimpleDispatchInfo::FixedOperational(20)]
+pub fn mission_critical_function() { some_sudo_op(); }
 ```
 
-> **Note:** Be careful! The default implementation of `SimpleDispatchInfo` resolves to
-> `FixedNormal(10_000)`. This is due to how things work in `substrate-node` and the desired
-> granularity of substrate. Even if you want to use the `SimpleDispatchInfo`, it is very likely that
-> you would want it to have a different `Default`.
-
-## Fee Calculation
+#### `MaxOperational`
 
-The final fee of a dispatch is calculated using the weight of the function and a number of
-configurable parameters.
+This is equivalent to `SimpleDispatchInfo::FixedOperational(Weight::max_value())`.
 
-### Inclusion Fee
+### Mandatory Dispatches
 
-A transaction fee consists of three parts:
+The most critical dispatches are classified as mandatory and represent dispatches that are part of
+the block validation process; they must be inherent, meaning they may not be signed. Mandatory
+dispatches will be included even if they cause a block to surpass the weight limit, so it is vital
+to ensure that they are validated separately. This can typically be accomplished by ensuring that
```

Comment:
I think this is true but missing the mark a little bit. Since they are inherents, they are not gossiped on the network or included in the tx queue. Only the block author puts them into a block. They will need custom validation logic so that other validators in the network can accept of reject the block.